CorePayments/src/main/java/com/paypal/android/corepayments/ResourceLoader.kt

```diff
@@ -0,0 +1,29 @@
+package com.paypal.android.corepayments
+
+import android.content.Context
+import android.content.res.Resources
+import androidx.annotation.RawRes
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import java.io.IOException
+
+class ResourceLoader(context: Context) {
```

Comment:
Can we add docstring or rename `ResourceLoader`? At a glance of the file standlone it isn't clear what it's used for